pymc/backends/arviz.py

```diff
@@ -43,10 +43,10 @@
 Var = Any  # pylint: disable=invalid-name
 
 
-def find_observations(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
+def find_observations(model: Optional["Model"]) -> Dict[str, Var]:
     """If there are observations available, return them as a dictionary."""
     if model is None:
-        return None
```

Comment:
We don't need the model is none cases. Whomever calls this function should always pass a model. There's no good reason why they would call the function with None